pkg/auth/ldaputil/query.go

```diff
@@ -239,6 +260,9 @@ func QueryForEntries(clientConfig *LDAPClientConfig, query *ldap.SearchRequest)
 	glog.V(4).Infof("searching LDAP server %v://%v at dn=%q with scope %v for %s requesting %v", clientConfig.Scheme, clientConfig.Host, query.BaseDN, query.Scope, query.Filter, query.Attributes)
 	searchResult, err := connection.Search(query)
 	if err != nil {
+		if strings.Contains(err.Error(), "LDAP Result Code 32") {
```

Comment:
They're not individually typed, and I'm not about to do a switch-case on `uint8` values of their `Error` field. I've made an upstream pull (https://github.com/go-ldap/ldap/pull/36) to make identification easier.